src/models/TimeTraveling.js

```diff
@@ -0,0 +1,216 @@
+import { observable } from "mobx"
+import { useComputed } from "./MobXHooks"
+import React, { useState, useRef } from "react"
+
+export const overrideDirectory = observable.box(null)
+global.ELECT_overrideDirectory = directory => overrideDirectory.set(directory)
+
+export function useDirectoryOverride() {
+  return [
+    useComputed(() => overrideDirectory.get(), []),
+    newDirectory => overrideDirectory.set(newDirectory),
+  ]
+}
+
+export function TimeMachine() {
+  const [override, setOverride] = useDirectoryOverride()
+  const [hover, setHover] = useState("")
+  const timeout = useRef()
+  return (
+    <div
+      data-active={override ? true : undefined}
+      css={{
+        textAlign: "center",
+        color: "#888",
+        "& > .status-text": { opacity: 0 },
+        "&:focus-within, &:focus, &:hover, &[data-active]": {
+          "& > .status-text": { opacity: 1 },
+        },
+      }}
+    >
+      <div className="status-text">
+        {hover && hover !== override ? (
+          <span>
+            <strong>Time machine:</strong> Time travel to {hover}
+          </span>
+        ) : override ? (
+          <span>
+            <strong>Time machine:</strong> Currently viewing {override}
+          </span>
+        ) : (
+          <span>
+            <strong>Time machine is inactive</strong> — viewing latest data
```

Comment:
What do you think if we change the labels to Thai?  Time travel might not be understood easily by general audience.